docs/whats-new.md

```diff
@@ -19,22 +19,36 @@
   </tbody>
 </table>
 
-## v3.1 (Planning/prototyping stage)
+## v4.0 (Planning/prototyping stage)
 
-Target Release Date: Q4 2021.
+Target Release Date: Q1 2022. Alpha releases available.
 
 Aspirational goals:
 
-- EcmaScript module support
+- EcmaScript module support (type: module).
 - Unbundled loaders.
 - More comprehensive support for `options.shape` to control output format of loaders.
 - Replace `Schema` class with arrow schema if arrowjs tree-shaking improvements are satisfactory.
 - New loaders: `GeoTIFFLoader`, `ZarrLoader`, `ParquetLoader`, `AVROLoader`.
 - `ffmpeg` WASM integration for `@loaders.gl/video`.
-- Upgrade basis to 1.15.
+
+## v3.1 (in testing)
+
+Target Release Date: Nov 2021.
+
+**@loaders.gl/textures** 
+
 - Parse KTX2 with BasisLoader.
```

Comment:
@belom88 Is this right? This is compressed texture loader? BasisLoader is deprecated?

Reply:
Both actual. BinomialLLC added support of KTX2 to basis_encoder.wasm. We can use it via CompressedTextureLoader or via BasisLoader directly.

Reply:
OK great. maybe you can clarify in a later PR?